client/homebrew/editor/stringArrayEditor/stringArrayEditor.jsx

```diff
@@ -109,35 +126,64 @@ const StringArrayEditor = createClass({
 		this.setState({ valueContext, updateValue: '' });
 	},
 
+	handleTagKeyDown : function(event, index) {
+		if(_.includes(['Enter', 'Space'], event.code)){
+			event.preventDefault();
+			this.editValue(index);
+			setTimeout(()=>{
+				event.target.querySelector('input').focus();
+			}, 0);
+		} else if(_.includes(['Delete'], event.key)){
+			this.removeValue(index);
+		} else if(_.includes(['ArrowLeft'], event.key)){
+			event.target.previousElementSibling?.focus();
+		} else if(_.includes(['ArrowRight'], event.key)){
```

Comment:
You can arrow-right all the way to the final text entry box, but you can't arrow-left from the text entry box back to the other tags.

Reply:
Can you try again?  I am able to use the arrow key to move from the final input to the previous tag without issue.

*However, you are correct if the final input has any text inside of it*-- the final input has to be cleared before you can use arrow keys to move out of it.  I'm pretty sure this was a conscious choice I made.  If someone wanted to move all the way to the beginning of their text using arrow keys, and held down the arrow key to do so, I wouldn't want to move them to the previous tag because they held down the key too long.   If necessary, I could detect if they are holding down the arrow key (rather than rapidly pressing it), and only prevent moving to the previous tag if true.  But I just didn't think it necessary at the time.

Reply:
> However, you are correct if the final input has any text inside of it-- the final input has to be cleared before you can use arrow keys to move out of it.

Ah, this is what I am seeing then. I was just playing around moving left and right and when I entered the final input I couldn't get back out.

I agree with your assessment that holding down the left arrow shouldn't fully leave the box. Maybe if you can detect that the cursor is in the first character position and only transition left in that case.